{
  "uid" : "1baafd4e22aafe2c",
  "name" : "Verify values of names and professions in specialist cards in the grid",
  "fullName" : "tests.specialists_page_test.TestSpecialistPageText#test_sp_02_04_verify_name_and_profession_in_cards",
  "historyId" : "78e03a13a1b27d1fe52cf41b87e50dab",
  "time" : {
    "start" : 1728784208618,
    "stop" : 1728784208618,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56080353f02a <unknown>\n#1 0x5608032255e0 <unknown>\n#2 0x56080321d7eb <unknown>\n#3 0x56080320da09 <unknown>\n#4 0x56080320f744 <unknown>\n#5 0x56080320dcbd <unknown>\n#6 0x56080320d52f <unknown>\n#7 0x56080320d422 <unknown>\n#8 0x56080320b493 <unknown>\n#9 0x56080320bada <unknown>\n#10 0x560803227f41 <unknown>\n#11 0x5608032b98c5 <unknown>\n#12 0x560803299b22 <unknown>\n#13 0x5608032b8d7d <unknown>\n#14 0x5608032998c3 <unknown>\n#15 0x5608032676b3 <unknown>\n#16 0x56080326868e <unknown>\n#17 0x560803509a2b <unknown>\n#18 0x56080350d9b1 <unknown>\n#19 0x5608034f6225 <unknown>\n#20 0x56080350e532 <unknown>\n#21 0x5608034db38f <unknown>\n#22 0x56080352df28 <unknown>\n#23 0x56080352e0f3 <unknown>\n#24 0x56080353de7c <unknown>\n#25 0x7fcd16a94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c0c5fabacb76da5493628a5e9c0d494d\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f6f3170>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x56080352e0f3 \\\\u003Cunknown>\\\\n#24 0x56080353de7c \\\\u003Cunknown>\\\\n#25 0x7fcd16a94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56080353f02a <unknown>\nE       #1 0x5608032255e0 <unknown>\nE       #2 0x56080321d7eb <unknown>\nE       #3 0x56080320da09 <unknown>\nE       #4 0x56080320f744 <unknown>\nE       #5 0x56080320dcbd <unknown>\nE       #6 0x56080320d52f <unknown>\nE       #7 0x56080320d422 <unknown>\nE       #8 0x56080320b493 <unknown>\nE       #9 0x56080320bada <unknown>\nE       #10 0x560803227f41 <unknown>\nE       #11 0x5608032b98c5 <unknown>\nE       #12 0x560803299b22 <unknown>\nE       #13 0x5608032b8d7d <unknown>\nE       #14 0x5608032998c3 <unknown>\nE       #15 0x5608032676b3 <unknown>\nE       #16 0x56080326868e <unknown>\nE       #17 0x560803509a2b <unknown>\nE       #18 0x56080350d9b1 <unknown>\nE       #19 0x5608034f6225 <unknown>\nE       #20 0x56080350e532 <unknown>\nE       #21 0x5608034db38f <unknown>\nE       #22 0x56080352df28 <unknown>\nE       #23 0x56080352e0f3 <unknown>\nE       #24 0x56080353de7c <unknown>\nE       #25 0x7fcd16a94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784208619,
      "stop" : 1728784209397,
      "duration" : 778
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784209398,
      "stop" : 1728784209820,
      "duration" : 422
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56080353f02a <unknown>\n#1 0x5608032255e0 <unknown>\n#2 0x56080321d7eb <unknown>\n#3 0x56080320da09 <unknown>\n#4 0x56080320f744 <unknown>\n#5 0x56080320dcbd <unknown>\n#6 0x56080320d52f <unknown>\n#7 0x56080320d422 <unknown>\n#8 0x56080320b493 <unknown>\n#9 0x56080320bada <unknown>\n#10 0x560803227f41 <unknown>\n#11 0x5608032b98c5 <unknown>\n#12 0x560803299b22 <unknown>\n#13 0x5608032b8d7d <unknown>\n#14 0x5608032998c3 <unknown>\n#15 0x5608032676b3 <unknown>\n#16 0x56080326868e <unknown>\n#17 0x560803509a2b <unknown>\n#18 0x56080350d9b1 <unknown>\n#19 0x5608034f6225 <unknown>\n#20 0x56080350e532 <unknown>\n#21 0x5608034db38f <unknown>\n#22 0x56080352df28 <unknown>\n#23 0x56080352e0f3 <unknown>\n#24 0x56080353de7c <unknown>\n#25 0x7fcd16a94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784209398,
        "stop" : 1728784209820,
        "duration" : 422
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56080353f02a <unknown>\n#1 0x5608032255e0 <unknown>\n#2 0x56080321d7eb <unknown>\n#3 0x56080320da09 <unknown>\n#4 0x56080320f744 <unknown>\n#5 0x56080320dcbd <unknown>\n#6 0x56080320d52f <unknown>\n#7 0x56080320d422 <unknown>\n#8 0x56080320b493 <unknown>\n#9 0x56080320bada <unknown>\n#10 0x560803227f41 <unknown>\n#11 0x5608032b98c5 <unknown>\n#12 0x560803299b22 <unknown>\n#13 0x5608032b8d7d <unknown>\n#14 0x5608032998c3 <unknown>\n#15 0x5608032676b3 <unknown>\n#16 0x56080326868e <unknown>\n#17 0x560803509a2b <unknown>\n#18 0x56080350d9b1 <unknown>\n#19 0x5608034f6225 <unknown>\n#20 0x56080350e532 <unknown>\n#21 0x5608034db38f <unknown>\n#22 0x56080352df28 <unknown>\n#23 0x56080352e0f3 <unknown>\n#24 0x56080353de7c <unknown>\n#25 0x7fcd16a94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"c0c5fabacb76da5493628a5e9c0d494d\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784209854,
      "stop" : 1728784209916,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 1,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 27,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "75da16e800287e43",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/75da16e800287e43",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f3830a502a <unknown>\n#1 0x55f382d8b5e0 <unknown>\n#2 0x55f382d837eb <unknown>\n#3 0x55f382d73a09 <unknown>\n#4 0x55f382d75744 <unknown>\n#5 0x55f382d73cbd <unknown>\n#6 0x55f382d7352f <unknown>\n#7 0x55f382d73422 <unknown>\n#8 0x55f382d71493 <unknown>\n#9 0x55f382d71ada <unknown>\n#10 0x55f382d8df41 <unknown>\n#11 0x55f382e1f8c5 <unknown>\n#12 0x55f382dffb22 <unknown>\n#13 0x55f382e1ed7d <unknown>\n#14 0x55f382dff8c3 <unknown>\n#15 0x55f382dcd6b3 <unknown>\n#16 0x55f382dce68e <unknown>\n#17 0x55f38306fa2b <unknown>\n#18 0x55f3830739b1 <unknown>\n#19 0x55f38305c225 <unknown>\n#20 0x55f383074532 <unknown>\n#21 0x55f38304138f <unknown>\n#22 0x55f383093f28 <unknown>\n#23 0x55f3830940f3 <unknown>\n#24 0x55f3830a3e7c <unknown>\n#25 0x7ff4ba694ac3 <unknown>",
        "time" : {
          "start" : 1728697322532,
          "stop" : 1728697322532,
          "duration" : 0
        }
      }, {
        "uid" : "abc0becd9f2e0f1a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/abc0becd9f2e0f1a",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56134ad3102a <unknown>\n#1 0x56134aa175e0 <unknown>\n#2 0x56134aa0f7eb <unknown>\n#3 0x56134a9ffa09 <unknown>\n#4 0x56134aa01744 <unknown>\n#5 0x56134a9ffcbd <unknown>\n#6 0x56134a9ff52f <unknown>\n#7 0x56134a9ff422 <unknown>\n#8 0x56134a9fd493 <unknown>\n#9 0x56134a9fdada <unknown>\n#10 0x56134aa19f41 <unknown>\n#11 0x56134aaab8c5 <unknown>\n#12 0x56134aa8bb22 <unknown>\n#13 0x56134aaaad7d <unknown>\n#14 0x56134aa8b8c3 <unknown>\n#15 0x56134aa596b3 <unknown>\n#16 0x56134aa5a68e <unknown>\n#17 0x56134acfba2b <unknown>\n#18 0x56134acff9b1 <unknown>\n#19 0x56134ace8225 <unknown>\n#20 0x56134ad00532 <unknown>\n#21 0x56134accd38f <unknown>\n#22 0x56134ad1ff28 <unknown>\n#23 0x56134ad200f3 <unknown>\n#24 0x56134ad2fe7c <unknown>\n#25 0x7f5572e94ac3 <unknown>",
        "time" : {
          "start" : 1728611013776,
          "stop" : 1728611013776,
          "duration" : 0
        }
      }, {
        "uid" : "aa953109720c28a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/aa953109720c28a",
        "status" : "passed",
        "time" : {
          "start" : 1728525002584,
          "stop" : 1728525002692,
          "duration" : 108
        }
      }, {
        "uid" : "379a918b3a60b896",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/379a918b3a60b896",
        "status" : "passed",
        "time" : {
          "start" : 1728438674580,
          "stop" : 1728438674696,
          "duration" : 116
        }
      }, {
        "uid" : "25350911921cabb6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/25350911921cabb6",
        "status" : "passed",
        "time" : {
          "start" : 1728352260506,
          "stop" : 1728352260630,
          "duration" : 124
        }
      }, {
        "uid" : "7bd303c0e45ef5b9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/7bd303c0e45ef5b9",
        "status" : "failed",
        "statusDetails" : "AssertionError: Names in specialist cards mismatch the valid values\nassert ['Login'] in [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]]\n +  where [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]] = SpecialistsPageData.specialists_names",
        "time" : {
          "start" : 1728266712425,
          "stop" : 1728266712474,
          "duration" : 49
        }
      }, {
        "uid" : "634722e35548a910",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/634722e35548a910",
        "status" : "passed",
        "time" : {
          "start" : 1728179803251,
          "stop" : 1728179803369,
          "duration" : 118
        }
      }, {
        "uid" : "2f5ed1b7c92404f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/2f5ed1b7c92404f",
        "status" : "passed",
        "time" : {
          "start" : 1728093044380,
          "stop" : 1728093044501,
          "duration" : 121
        }
      }, {
        "uid" : "4ea350e485792df4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/4ea350e485792df4",
        "status" : "passed",
        "time" : {
          "start" : 1728006746996,
          "stop" : 1728006747106,
          "duration" : 110
        }
      }, {
        "uid" : "dc9500c785bd4008",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/dc9500c785bd4008",
        "status" : "passed",
        "time" : {
          "start" : 1727920392418,
          "stop" : 1727920392531,
          "duration" : 113
        }
      }, {
        "uid" : "b1c8d7914226bcf0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/b1c8d7914226bcf0",
        "status" : "passed",
        "time" : {
          "start" : 1727833812717,
          "stop" : 1727833812838,
          "duration" : 121
        }
      }, {
        "uid" : "eea477ae303cc5b2",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/eea477ae303cc5b2",
        "status" : "passed",
        "time" : {
          "start" : 1727747937364,
          "stop" : 1727747937478,
          "duration" : 114
        }
      }, {
        "uid" : "dcfb35cf93e32896",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/dcfb35cf93e32896",
        "status" : "passed",
        "time" : {
          "start" : 1727661219210,
          "stop" : 1727661219323,
          "duration" : 113
        }
      }, {
        "uid" : "615f45dde4850b13",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/615f45dde4850b13",
        "status" : "passed",
        "time" : {
          "start" : 1727575040169,
          "stop" : 1727575040284,
          "duration" : 115
        }
      }, {
        "uid" : "94568d25c66f22af",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/94568d25c66f22af",
        "status" : "passed",
        "time" : {
          "start" : 1727488266872,
          "stop" : 1727488266989,
          "duration" : 117
        }
      }, {
        "uid" : "1b96fc6012e4db7c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/1b96fc6012e4db7c",
        "status" : "passed",
        "time" : {
          "start" : 1727402014560,
          "stop" : 1727402014679,
          "duration" : 119
        }
      }, {
        "uid" : "7d2e32ffdc027402",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/7d2e32ffdc027402",
        "status" : "passed",
        "time" : {
          "start" : 1727315488509,
          "stop" : 1727315488625,
          "duration" : 116
        }
      }, {
        "uid" : "d7bab9c8bb4f449c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/d7bab9c8bb4f449c",
        "status" : "passed",
        "time" : {
          "start" : 1727229135301,
          "stop" : 1727229135421,
          "duration" : 120
        }
      }, {
        "uid" : "10c9b8c91c4d08ef",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/10c9b8c91c4d08ef",
        "status" : "passed",
        "time" : {
          "start" : 1727142666675,
          "stop" : 1727142666790,
          "duration" : 115
        }
      }, {
        "uid" : "5e129b654c18f332",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/5e129b654c18f332",
        "status" : "passed",
        "time" : {
          "start" : 1727056186712,
          "stop" : 1727056186825,
          "duration" : 113
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1baafd4e22aafe2c.json",
  "parameterValues" : [ ]
}